tensornetwork/tensornetwork_test.py

```diff
@@ -1165,6 +1165,8 @@ def test_switch_backend():
 
 
 def test_svd_consistency(backend):
+  if backend == "pytorch":
+    pytest.skip()
```

Comment:
Be sure to add "PyTorch doesn't support complex numbers" as the skip reason.

Reply:
Done!